src/docs/truffle/getting-started/debugging-your-contracts.md

```diff
@@ -107,6 +127,21 @@ however, the debugger can likely avoid the initial recompile, speeding up
 startup greatly.
 </p>
 
+## Debugging external contracts with verified source
+
+If you pass the `--fetch-external` option, the debugger will attempt to download verified source code off of Etherscan for any addresses involved in the transaction that it cannot find source code for in your project.  (Support for retrieving code from Sourcify as well is planned.)  You can of course debug such transactions without this option, but when stepping through the transaction the external calls to these unrecognized contracts will simply be skipped over.
+
+This option can also be abbreviated `-x`.
+
+If you have an Etherscan API key, you can include it in your configuration file and the debugger will use it when downloading source from Etherscan.  Including this can speed up downloads.
+
+Example:
+```javascript
+etherscan: {
+  apiKey: "0123456789abcdef0123456789abcdef" //replace this with your API key if you have one
+}
+```
```

Comment:
Let's wrap this with contextual info:

```suggestion
```javascript
module.exports = {
  /* ... rest of truffle-config.js ... */
  etherscan: {
    apiKey: "0123456789abcdef0123456789abcdef" //replace this with your API key if you have one
  }
}
```
```

Reply:
Good idea!